CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+- [Pre-built images](#pre-built-images)
+- [Build Requirements](#build-requirements)
+- [Building Executables and Images from Scratch](#building-executables-and-images-from-scratch)
+- [Development](#development)
+- [Local Testing](#local-testing)
+
+## Pre-built images
+
+For a list of supported public images of Tornjak along with usage instructions please see our [USAGE document](./USAGE.md).
+
+## Build Requirements
+
+In order to build from scratch, we require the following installations:
```

Comment:
Let's not use the expression "to build from scratch" as this is ambiguous and it might imply we use `Scratch` Docker image as a foundation for our Tornjak images, but unfortunately this is not a case. :-)